projects/scripts/adi_board.tcl

```diff
@@ -335,6 +335,199 @@ proc ad_xcvrpll {m_src m_dst} {
   }
 }
 
+###################################################################################################
```

Comment:
Would be nice to put this process in a separate file, adi_board.tcl should contain generic processes which are used for block design creation.

Reply:
Done.